src/org/rascalmpl/uri/file/SystemPathURIResolver.java

```diff
@@ -30,6 +31,9 @@ public class SystemPathURIResolver implements ILogicalSourceLocationResolver {
 	public String scheme() {
 		return "PATH";
```

Comment:
also, why is this in capitals?

Reply:
To communicate that this is literally reflection of the PATH environment variable.

Reply:
On windows it's capitalized as `Path`, and it's case-insensitive :) Aren't our schema's case-insensitive?

Anyway, it look a bit strange to me. Not a hard opinion, just something I noticed.

Reply:
I'm more worried about the memory usage. It's not like we aren't using way too much memory already. Is it really necessary to add another cache?

Reply:
okay, I could remove that cache, just seemed a bit wastful to me. but I've not profiled it, so maybe safer not to add a cache.